Sources/Misc/DangerousSettings.swift

```diff
@@ -100,3 +97,16 @@ import Foundation
 }
 
 extension DangerousSettings: Sendable {}
+
+/// Dangerous settings not exposed outside of the SDK.
+internal protocol InternalDangerousSettingsType: Sendable {
```

Comment:
This allows having a custom type in integration tests so we can swap the return value, without having to make the actual `DangerousSettings.Internal` `struct` mutable.